.github/workflows/package_installers.yml

```diff
@@ -2,7 +2,7 @@ name: Package Installers
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, issue/1200/sign-mac-os ]
```

Comment:
Do we still need `issue/1200/sign-mac-os` here? I assume it was for testing purposes, but I may be wrong?

Reply:
That was added for testing and is no longer needed.  I have removed it.